charts/open-notificaties/templates/deployment.yaml

```diff
@@ -133,5 +139,79 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-
-
+---
+{{- if .Values.flower.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "open-notificaties.flowerFullname" . }}
+  labels:
+  {{- include "open-notificaties.flowerLabels" . | nindent 4 }}
+spec:
+  {{- if not .Values.flower.autoscaling.enabled }}
+  replicas: {{ .Values.flower.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "open-notificaties.flowerSelectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+        {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+    {{- include "open-notificaties.flowerSelectorLabels" . | nindent 8 }}
+    {{- with .Values.flower.podLabels }}
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "open-notificaties.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-flower
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          envFrom:
+            - secretRef:
+                name: {{ .Values.flower.existingSecret | default (include "open-notificaties.fullname" .) }}
```

Comment:
I see Flower has its own `existingSecret` while it defaults to the secret of Open Notificaties. I would expect that Flower has its own Secret here.

Reply:
Fixed in 762efaeec4135ab957ea7078b4dfe2cefa0e77ed.

Reply:
Looks good to me 👍